man/ocf_suse_SAPHanaTopology.7

```diff
@@ -1,6 +1,6 @@
-.\" Version: 0.155.0
+.\" Version: 0.181.0
```

Comment:
are you sure with this version string for SAPHanaSR ScaleUp?

Reply:
Hi Angela,

the man pages SAPHanaSR_maintenance_examples.7 and ocf_suse_SAPHanaToplogy.7 are identical for scale-up and scale-out. So I tried keeping the version string indentical. Of course it is wrong then for one package.
Should we put both versions in?

Regards,
Lars

Reply:
No, please. One version is absolute sufficient. And this is the version of the man page and not of the package, even that  we sometimes try to unify both versions. But that's not mandatory.
So it's okay from my side to have version '0.181.0' in the man page. It was only the question, if the version bump was done by intention. 
So all fine and I will approve and merge the change.
Thanks for the effort.